ql/src/test/queries/clientpositive/udf_array_intersect.q

```diff
@@ -0,0 +1,42 @@
+--! qt:dataset:src
+
+-- SORT_QUERY_RESULTS
+
+set hive.fetch.task.conversion=more;
+
+DESCRIBE FUNCTION array_intersect;
+DESCRIBE FUNCTION EXTENDED array_intersect;
+
+-- evalutes function for array of primitives
+SELECT array_intersect(array(1, 2, 3, null,3,4),array(1, 3, null)) FROM src tablesample (1 rows);
+
+SELECT array_intersect(array(),array()) FROM src tablesample (1 rows);
+
+SELECT array_intersect(array(null),array(null)) FROM src tablesample (1 rows);
+
+SELECT array_intersect(array(1.12, 2.23, 3.34, null,1.11,1.12,2.9),array(1.12,3.34,1.11,1.12)) FROM src tablesample (1 rows);
+
+SELECT array_intersect(array(1.1234567890, 2.234567890, 3.34567890, null, 3.3456789, 2.234567,1.1234567890),array(1.1234567890, 3.34567890, null,2.234567)) FROM src tablesample (1 rows);
+
+SELECT array_intersect(array(11234567890, 2234567890, 334567890, null, 11234567890, 2234567890, 334567890, null),array(11234567890, 2234567890, 334567890)) FROM src tablesample (1 rows);
```

Comment:
I get the array_intersect part and it's implementation. But what I don't understand is ` FROM src tablesample (1 rows);` -- Why do we have to perform intersect on a table 'src'?